spec/lib/archiving_spec.rb

```diff
@@ -76,4 +84,31 @@
     s2 = Archiving.serialise(delivery)
     expect(s1).to eq s2
   end
+
+  describe ".archive" do
+    around do |example|
+      # Silence debugging output from this method
+      silence_stream(STDOUT) do
+        example.run
+      end
+    end
+
+    it "removes the temp archive file it creates" do
+      delivery = Delivery.create!(
+        id: 5,
```

Comment:
Do we really need to set all of these specific attributes for this test? Would this be better created by FactoryGirl anyway?